src/components/dashboard/HeaderDasboard.tsx

```
@@ -85,31 +85,46 @@ export default function HeaderDasboard() {
+            </button>
             <Link href="/" className="text-xl font-semibold text-orange-700 hover:text-orange-900">
               ALIMANA
             </Link>
           </div>
 
           {/* Header right */}
           <div className="flex items-center space-x-4">
-            <button className="text-orange-700 relative">
-              <Bell className="w-5 h-5" />
-              <span className="absolute top-0 right-0 h-2 w-2 rounded-full bg-red-500" />
+            <button onClick={toggleDarkMode} title="Toggle dark mode">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)